.travis.yml

```diff
@@ -2,9 +2,13 @@ language: scala
 matrix:
   include:
     - scala: 2.11.12
-      jdk: openjdk8
+      jdk:
+      - openjdk8
+      - oraclejdk11
     - scala: 2.12.7
-      jdk: openjdk8
+      jdk:
+      - openjdk8
+      - oraclejdk11
```

Comment:
Hi @andrewsmartin stumbled on his; I think it would be better to use `openjdk11`

Reply:
hey @regadas, I'm happy to switch (no strong opinion on my side), but just curious about `would be better to use openjdk11`, can you elaborate? FWIW, I was using oraclejdk8 previously until Travis CI stopped supporting that, so for 8 I switched to open.

Reply:
Hey @andrewsmartin, no special reason really;  by using `openjdk11` you might avoid having the same CI issue in the future.